1-exercises/B-while-loop/exercise.js

```diff
@@ -6,7 +6,19 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+  i = [];
+  x = 0;
+  while (x < n) {
+    // console.log(x, n);
+    let newEven = x * 2;
+    i.push(newEven); //add element to the array
+    x++;
+
+    //console.log(newEven);
+  }
+  // console.log(i);
+
+  console.log(i.join()); //convert to string  comma-seperated
```

Comment:
Hello Iwona! great way using join well done. I have few suggestions : 

 let is  missing on line 9 and 10
It is a good practice to give a name to variable that explaining what that variable is being used at any time. 
on line 13 

function evenNumbers(n) {
  let counter = 0;
  let evenNum = 0;
  while (counter < n) {
    console.log(evenNum);
    evenNum += 2
    counter++;
  }
}